analyzer/src/funTest/kotlin/managers/CocoaPodsFunTest.kt

```diff
@@ -27,7 +27,7 @@ import java.time.Instant
 
 import org.ossreviewtoolkit.downloader.VersionControlSystem
 import org.ossreviewtoolkit.model.ProjectAnalyzerResult
-import org.ossreviewtoolkit.utils.core.Os
+import org.ossreviewtoolkit.utils.common.Os
```

Comment:
Commit message: Maybe say that the log statement was dropped for simplicity as the log helper is not available, and because the (eventually fixuped) user home it simplicitly dumped as part of printing `ORT_DATA_DIR`.